modules/omx/omx.c

```diff
@@ -0,0 +1,300 @@
+/**
+ * @file omx.c     Raspberry Pi VideoCoreIV OpenMAX interface
+ *
+ * Copyright (C) 2016 - 2017 Creytiv.com
+ * Copyright (C) 2016 - 2017 Jonathan Sieber
+ */
+
+#include "omx.h"
+
+#include <re/re.h>
+#include <rem/rem.h>
+#include <baresip.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
```

Comment:
no need to include assert.h as the code should not use assert()
(only in extremely rare cases).